experimental/regression_suite/ireers/fixtures.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+
+from typing import Dict, Sequence, Union
+from pathlib import Path
+import subprocess
+import time
+
+from .artifacts import (
+    Artifact,
+    FetchedArtifact,
+    ProducedArtifact,
+)
+
+
+IREE_COMPILE_QOL_FLAGS = [
+    "--mlir-timing",
+    "--mlir-timing-display=tree",
```

Comment:
https://github.com/openxla/iree/actions/runs/5971191173/job/16199980567#step:6:6677 6000 lines of output from this 🤔